.evergreen/evergreen.yml.in

```
@@ -1365,6 +1391,36 @@ buildvariants:
       <% } } } %>
       - name: sign_artifact_win32_x64
       - name: sign_artifact_win32msi_x64
+  - name: verify_debian_artifact
+    display_name: "Ubuntu 20.04 x64 (Verification)"
```

```suggestion
    display_name: "Ubuntu 20.04 x64 (Signature Verification)"
```

(or the suggestion below this one, I have a mild preference for that one but either way is fine by me)